plugins/pulp_rpm/plugins/importers/iso/sync.py

```diff
@@ -160,6 +160,7 @@ def download_succeeded(self, report):
                     iso.save()
                 except NotUniqueError:
                     iso = iso.__class__.objects.filter(**iso.unit_key).first()
+                self.add_catalog_entries([iso])
```

Comment:
Will the workflow get to here if the download strategy is on_demand or background? If this is only hit when the strategy is immediate, do we actually want to add catalog entries?